<launch>
    <!-- Hardware Interface -->
    <node name='rover_4_base' type='hw_interface_node2' pkg='rover_4_core'>
        <remap from="/rover_4_base/cmd_vel" to="/cmd_vel"/>
        <remap from="/rover_4_base/odom" to="/odom"/>
    </node>
    <!-- Load controller config -->
    <rosparam command="load" 
              file="$(find rover_4_core)/hw_interface/config/common.yaml"/>
   
    <node name="controller_spawner" 
          pkg="controller_manager" 
          type="spawner" 
          output="screen" 
          args="rover_4_base"/> <!-- joint_state_controller leftWheel_effort_controller rightWheel_effort_controller -->

    
</launch>